target/linux/realtek/dts-5.15/rtl838x.dtsi

```diff
@@ -117,8 +117,14 @@
 		};
 	};
 
+	aliases {
+		serial0 = &uart0;
+		serial1 = &uart1;
+	};
+
 	chosen {
-		bootargs = "console=ttyS0,115200";
+		bootargs = "earlycon";
+		stdout-path="serial0:115200n8";
```

Comment:
A bit of nit, but please leave spaces around " = ". I think you have it the same in all the changes.

Reply:
> A bit of nit, but please leave spaces around " = ". I think you have it the same in all the changes.

Ohhh that I will change asap :D